contracts/contracts/vault/VaultStorage.sol

```diff
@@ -165,8 +178,41 @@ contract VaultStorage is Initializable, Governable {
     }
     SwapConfig internal swapConfig = SwapConfig(address(0), 0);
 
+    /// @notice Address of the Dripper contract that streams harvested rewards to the Vault
+    /// @dev The vault is proxied so needs to be set with setDripper against the proxy contract.
+    // slither-disable-next-line uninitialized-state
+    address public dripper;
+
+    /// Withdrawal Queue Storage /////
+
+    struct WithdrawalQueueMetadata {
+        // cumulative total of all withdrawal requests included the ones that have already bene claimed
+        uint128 queued;
+        // cumulative total of all the requests that can be claimed included the ones that have already bene claimed
+        uint128 claimable;
+        // total of all the requests that have been claimed
+        uint128 claimed;
+        // index of the next withdrawal request starting at 0
+        uint128 nextWithdrawalIndex;
+    }
+
+    WithdrawalQueueMetadata public withdrawalQueueMetadata;
+
+    struct WithdrawalRequest {
+        address withdrawer;
+        bool claimed;
+        // Amount of oTokens to redeem
+        uint128 amount;
+        // cumulative total of all withdrawal requests including this one.
+        // this request can be claimed when this queued amount is less than or equal to the queue's claimable amount.
+        uint128 queued;
+    }
+
+    // Mapping of withdrawal requests indexes to the user withdrawal request data
+    mapping(uint256 => WithdrawalRequest) public withdrawalRequests;
+
     // For future use
-    uint256[50] private __gap;
+    uint256[46] private __gap;
```

Comment:
I see three slots added? `dripper`, `withdrawalQueueMetadata`, and  `withdrawalRequests`. Shouldn't the gap change by 3, not four?

This particular gap is one that keeps me up at night. We should make sure our fork tests will catch an incorrect number here.

Reply:
There's three storage variables added but `withdrawalQueueMetadata` is a struct that takes up 2 slots hence the gap was reduced by 4 slots.